.gitignore

```diff
@@ -5,3 +5,5 @@ coverage/
 .idea
 .sass_cache
 _site
+.vscode
```

Comment:
The `.idea` one was a mistake that got merged at some point. Let's not make more of those.
Please put these lines in your global `.gitignore` instead.